web-registry/src/components/atoms/BreakText.tsx

```diff
@@ -0,0 +1,6 @@
+import { styled } from '@mui/material';
+
+export const BreakText = styled('div')({
```

Comment:
Should we styled a `Box` instead of `div` to make it customizable with `sx` if we need to?
Like we do for `Center` in `Box.tsx` file.

Same for BreakTextEnd and GreyText.